core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -81,74 +81,75 @@
   }
 
 </style>
+
```

Comment:
Is it an expected change?

Reply:
no, it wasn't there in the main repo, I have removed the extra line.